src/ngraph_builder.cc

```diff
@@ -2252,7 +2253,7 @@ static Status TranslateGatherV2Op(
   // Negative axis is supported. Accounting for that
   auto ng_input_shape = ng_input->get_shape();
   size_t ng_input_rank = ng_input_shape.size();
-  int axis;
+  size_t axis;
```

Comment:
Minor:
Since we are changing `axis` to `size_t` from `int`, the check that follows `if (axis < 0 || axis >= ng_input_rank)` can be `if (axis >= ng_input_rank)`